packages/web-wallet/src/components/views/CredentialsList/index.tsx

```diff
@@ -25,22 +29,33 @@ const CredentialsList: FC<Props> = (props: Props): ReactElement => {
     isLoading: credentialsLoading,
     isError: credentialsError,
     refetch: refetchCredentials,
-  } = useList<Credential, HttpError>({
-    resource: 'credential',
-    dataProviderName: 'supaBase',
+  } = useList<DigitalCredential, HttpError>({
+    resource: DataResource.CREDENTIALS,
     pagination: {
       pageSize: 1000,
       mode: 'server',
     },
     sorters: [
       {
-        field: 'issuanceDate',
+        field: 'createdAt', // We don't we have a field 'issuanceDate' in the db anymore, so sort on createdAt
```

Comment:
Whut we must have a field that indicates the actual issuance date. It is called validFrom in VCDM 2.0. createdAt is the DB's notion of when an entity is stored.

Reply:
Can't a credential be created first, then after some approval process issued next day, then be valid starting some time in the future?
I can take validFrom for now but it does not seem the same as issuanceDate. (It's the overview list, so it does not seem super critical for now.)